builtin/game/chatcommands.lua

```diff
@@ -718,7 +718,7 @@ core.register_chatcommand("rollback", {
 	description = "Revert actions of a player. Default for <seconds> is 60",
 	privs = {rollback=true},
 	func = function(name, param)
-		if not core.setting_getbool("enable_rollback_recording") then
+		if not core.settings:get_bool("enable_rollback_recording") then
```

Comment:
you renamed getbool to get_bool

Consistency is more important than a long name